roles/etcd/defaults/main.yaml

```diff
@@ -14,12 +13,12 @@ etcd_peer_cert_file: "{{ etcd_conf_dir }}/peer.crt"
 etcd_peer_key_file: "{{ etcd_conf_dir }}/peer.key"
 
 etcd_initial_cluster_state: new
-etcd_initial_cluster_token: etcd-cluster-1
+etcd_initial_cluster_token: etcd-os-cluster-1
```

Comment:
I'll just revert that change for this PR, if we need more flexibility we can change.  But I don't think most people will care.